GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTCOREvent+GDTCCTSupport.m

```diff
@@ -158,28 +175,33 @@ - (NSNumber *)eventCode {
   return nil;
 }
 
-- (void) setEventCode:(NSNumber *)eventCode {
+- (void)setEventCode:(NSNumber *)eventCode {
+  if (eventCode == nil) {
```

Comment:
if eventCode is nil, I'd expect it to be removed from the customBytes JSON dictionary, not be a no-op